oasislmf/pytools/gul/common.py

```diff
@@ -11,15 +11,22 @@
 # probably need to set this dynamically depending on the stream type
 gul_header = np.int32(1 | 2 << 24).tobytes()
 
-PIPE_CAPACITY = 65536  # bytes
-GETMODEL_STREAM_BUFF_SIZE = 2 * PIPE_CAPACITY
-
 items_data_type = nb.from_dtype(np.dtype([('item_id', np.int32),
                                           ('damagecdf_i', np.int32),
                                           ('rng_index', np.int32)
                                           ]))
 
-coverage_type = nb.from_dtype(np.dtype([('tiv', np.float),
+# TODO: 'vulnerability_id' should be renamed to 'vulnerability_idx' because it is the
```

Comment:
I recently changed my mind on TODOs. I think we should avoid them in the code and create separate issue of what needs to be done.

Reply:
actually, the name is correct. in weighted vulnerability we're going to store again the actual vulnerability_id in that field, so no need to rename it.